deploy/011-initial-liquidity.ts

```diff
@@ -0,0 +1,66 @@
+import { ethers } from "hardhat";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { DeploymentConfig, PoolConfig, TokenConfig, getConfig, getTokenConfig } from "../helpers/deploymentConfig";
+import { getUnregisteredVTokens } from "../helpers/deploymentUtils";
+import { ERC20, MockToken } from "../typechain";
+
+const getUnderlyingMock = async (assetSymbol: string): Promise<MockToken> => {
+  return ethers.getContract<MockToken>(`Mock${assetSymbol}`);
+};
+
+const getUnderlyingToken = async (assetSymbol: string, tokensConfig: TokenConfig[]): Promise<ERC20> => {
+  const token = getTokenConfig(assetSymbol, tokensConfig);
+  let underlyingAddress = token.tokenAddress;
+  if (token.isMock) {
+    underlyingAddress = (await getUnderlyingMock(assetSymbol)).address;
+  }
+  return ethers.getContractAt<ERC20>("@openzeppelin/contracts/token/ERC20/ERC20.sol:ERC20", underlyingAddress);
+};
```

Comment:
These two functions are already defined (in a very similar way) and used in 013. Just in case you would like to consider moving them to the deploymentUtils.ts file :)